packages/nextjs/components/onchain-impact-dashboard/projectTotalsGraph/projectTotalsGraph.tsx

```diff
@@ -32,41 +33,48 @@ export const ProjectTotalsGraph = ({
   totalsRecord,
   filter,
 }: IProps) => {
-  const formatNumber = (num: number, dec?: number) => {
-    const m = 1000000;
-    const k = 1000;
-    const text = num >= m ? "M" : num >= k ? "K" : "";
-    if (dec != undefined) {
-      const divisor = num >= m ? m : num >= k ? k : 1;
-      const value = new Intl.NumberFormat().format(parseFloat((num / divisor).toFixed(dec)));
-      return `${value}${text}`;
-    }
-    return new Intl.NumberFormat().format(num);
-  };
-  const CustomTooltip = ({ active, payload, label }: any) => {
-    if (active && payload && payload.length) {
-      return (
-        <div className="bg-base-300 p-4 rounded-lg shadow">
-          <p className="m-0 p-0 label font-bold">{formatDate(label)}</p>
-          {payload.map((entry: any, index: any) => (
-            <div key={`item-${index}`}>
-              <p style={{ color: entry.color }} className="m-0 p-0 label">
-                {formatNumber(entry.value)}
-              </p>
-            </div>
-          ))}
-        </div>
-      );
-    }
-    return null;
+  const options = {
+    chart: {
+      type: "line",
+      height: 360,
+    },
+    title: {
+      text: "",
+    },
+    subtitle: {
+      text: "",
+    },
+    credits: {
+      enabled: false,
+    },
+    xAxis: {
+      type: "datetime",
+      title: {
+        text: "",
+      },
+      dateTimeLabelFormats: {
+        day: "%e %b",
+      },
+    },
+    legend: {
+      enabled: false,
+    },
   };
+
+  const chartComponentRef = useRef<HighchartsReact.RefObject>(null);
   const metricToWork = metrics[selectedMetric];
   const values = totalsRecord.map(d => d[metricToWork?.name]);
+  const newValues = totalsRecord.map(d => {
+    const date = new Date(d["date"]);
+    return [Date.UTC(date.getFullYear(), date.getMonth(), date.getDate()), d[metricToWork?.name]];
```

Comment:
In Jonas' feedback in #91 he said he wanted the hover overlay to show the date as Apr 1, 2024 and this PR still has it showing the day of the week as well.
![image](https://github.com/user-attachments/assets/02931d9f-438d-48c8-bb1b-0ed709ece83b)

Reply:
It is fine to leave as is for this PR but we just shouldn't close #91 until this has been addressed.

Reply:
Also worth noting we may need to do some formatting on larger numbers. I feel like a comma might make this look better. 
![image](https://github.com/user-attachments/assets/fbaa26da-d7af-4c8b-92c0-7a0fdccec35e)

Reply:
done 
<img width="234" alt="Captura de pantalla 2024-07-24 a las 21 14 11" src="https://github.com/user-attachments/assets/96b42d9f-8149-44b4-99b3-82bd64c83236">